custom_components/waste_collection_schedule/waste_collection_schedule/source/aw_lkspn_de.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+
+# URL, DESCRIPTION  and TITLE of the website
+TITLE = "Eigenbetrieb Abfallwirtschaft Landkreis Spree-Neiße"
+DESCRIPTION = "Source for Eigenbetrieb Abfallwirtschaft Landkreis Spree-Neiße."
+URL = "https://www.eigenbetrieb-abfallwirtschaft.de"
+COUNTRY = "de"
+TEST_CASES = {
+    "Forst (Lausitz), Rosenweg": {"city": "4", "street": "344"},
+    "Peitz, Am See": {"city": "8", "street": "1077"},
+    "Guben, Altsprucke": {"city": "5", "street": "410"},
```

Comment:
Please make one test case all Ints as the yaml input of integers without `"` will lead to an integer argument, which should still work as users should not care what type their argument has. (lead to problems in the past (e.g. #1935))

Reply:
Added a test with integer input. Test run successful ...